translations/translation_br.xml

```diff
@@ -618,50 +616,27 @@ Para excluir uma rota ou pasta, clique no botão excluir e depois na pasta ou ro
-"/>
-    <text name="CP_help_page_minihud_balecollect_title" text="Fardos coletar/embalar"/>
-    <text name="CP_help_page_minihud_balecollect_text" text="
-A: Fardos restantes no campo.
-B: Tipo de fardos a recolher/embalar.
-C: Deslocamento entre a linha central do trator e a linha central do braço da carregadeira. Pode ser necessário ajustar isso para tratores maiores (por exemplo, com pneus mais largos)."/>
-    <text name="CP_help_page_minihud_siloloader_title" text="Carregar silo"/>
-    <text name="CP_help_page_minihud_siloloader_text" text="
-A: Tamanho restante da pilha em litros.
-B: Largura de trabalho igual à do trabalho de campo.
-C: O Courseplay precisa da altura exata acima do solo definida corretamente. Como esta altura pode ser diferente para cada ferramenta, você pode verificá-la e ajustá-la com esta configuração.
-D: Assim como no descarregador, você pode copiar as posições dos marcadores para outro veículo.
-"/>
-    <text name="CP_help_page_minihud_siloworker_title" text="Trabalho no silo"/>
-    <text name="CP_help_page_minihud_siloworker_text" text="
-R: Similar ao carregar de silo, a altura do nivelador é crítica. Você pode ajustá-lo aqui.
-B: Mostra o andamento da compactação. Clicando nele, alterna a opção de parar o driver quando atingir a compactação total.
-C: Você pode alterar a largura de trabalho quando necessário aqui.
-D: Esta configuração informa ao trabalhador para esperar no silo ou em uma posição de estacionamento selecionada quando um descarregador se aproxima do silo.
+    <text name="CP_help_page_minihud_text" text="
+R: Clicar no nome do veículo abrirá o menu de configuração do veículo.
+B: Seleciona o ponto inicial onde você deseja que o auxiliar comece.
+C: Clicar no nome do curso abrirá o menu AI.
+     Se o veículo estiver em um campo, as configurações do gerador de percurso também serão abertas.
+D: Inicia ou para o auxiliar. Se a cor for verde, o auxiliar CP está ativo.
+E: Mostra o waypoint atual/total de waypoints. Clicar nele abrirá o menu do gerenciador do curso.
+F: Clicar neste ícone abre o menu de configurações globais.
+G: Fecha o HUD.
+H: Diminuir ou aumentar a largura de trabalho. Uma linha vermelha no seu veículo mostra o valor atual.
+I: Valor para deslocamento horizontal, + esquerda, - direita.
+J: Copie e cole um percurso em um veículo diferente.
+     Depois que o botão copiar for pressionado, o percurso poderá ser colado em um veículo sem percurso quando o ícone ficar verde.
+K e L: Clicar no texto redefinirá suas alterações para os valores padrão.
+M: Mostra o curso copiado.
+N: Esta área mostra o deslocamento da pista se você tiver um curso de múltiplas ferramentas carregado para o comboio. 
+     Clicar no texto alterna as posições.
+O: Iniciar/parar a gravação de campos personalizados.
+P: Remove o percurso do veículo.
+P: Altera a visibilidade do percurso entre: ligado, desligado e apenas início/parada.
+R: Remove o curso copiado da área de transferência.
```

Comment:
Same for this one. Check from this line. https://github.com/Courseplay/Courseplay_FS22/blob/ab4e01d97a9c2f78e5b2a27fdb75ecb312f00144/translations/translation_en.xml#L623